python/mlc_llm/quantization/group_quantization.py

```diff
@@ -918,6 +914,7 @@ def from_mixtral_experts(
         if "shard_strategy" in src.weight.attrs:
             shard = src.weight.attrs["shard_strategy"]
             apply_sharding(shard, f"{shard.name}_q_weight", quantized_mistral_experts.q_weight)
+            # TODO(@team): upstream does not use this condition
```

Comment:
cc. @csullivan @JosephTheOctonaut @vinx13

Reply:
can remove this, it is also fp8